.coveragerc

```diff
@@ -25,6 +25,7 @@ omit =
     homeassistant/components/adguard/switch.py
     homeassistant/components/ads/*
     homeassistant/components/aftership/sensor.py
+    homeassistant/components/agent_dvr/*
```

Comment:
Tests with 100% coverage for the config_flow need to be provided, and thus should not be excluded from coverage here either.